controllers/replication-controller/dellcsireplicationgroup_controller.go

```diff
@@ -478,3 +516,340 @@ func (r *ReplicationGroupReconciler) SetupWithManager(mgr ctrl.Manager, limiter
 		}).
 		Complete(r)
 }
+
+// Give a replication group name and target, swapAllPVC reassigns the PVC from local volume to remote volume.
+// It also retains the original reclaimPolicy and operates within a single cluster.
+func (r *ReplicationGroupReconciler) swapAllPVC(ctx context.Context, c connection.RemoteClusterClient, rgName string, rgTarget string, log logr.Logger) error {
+	log.V(common.InfoLevel).Info(fmt.Sprintf("calling getPVList from %s\n", rgName))
+	pvcs, err := c.ListPersistentVolumeClaim(ctx, client.MatchingLabels{r.Domain + "/replicationGroupName": rgName})
```

Comment:
**suggestion:** We should be able to use the `controller.ReplicationGroup` from the `common.go` for the labels.